packages/attestation-service/src/db.ts

```diff
@@ -15,34 +15,36 @@ export function initializeDB() {
 
 export let kit: ContractKit
 
-export function initializeKit() {
+export async function initializeKit() {
   if (kit === undefined) {
     kit = newKit(fetchEnv('CELO_PROVIDER'))
+    const blockNumber = await kit.web3.eth.getBlockNumber()
+    if (blockNumber === 0) {
```

Comment:
Is it sufficient to know the blockNumber isn't 0 or should we wait for web3 to be synced?

Reply:
We should have proper waiting for sync, raised https://github.com/celo-org/celo-monorepo/issues/1742 for it